### DISCUSSION TITLE
Is there a way to serve a Ghost blog on a /blog path in my Next JS application?

### DISCUSSION THREAD

**Discussion by razvanilin** [vercel/vercel#5804] [Help]
Hi there,

I'm starting this discussion here because google searches didn't work so far.

My current setup:

* chartbrew.com - NextJS app hosted on Vercel (Cloudflare DNS with a CNAME pointing to Vercel)
* blog.chartbrew.com - Ghost blog hosted on my server (DNS and proxied with Cloudflare)

So my plan is to manage to "move" my blog from a subdomain to a `/blog` path in my NextJS app.

* `blog.chartbrew.com` -> `chartbrew.com/blog`

I tried following [this guide](https://hackernoon.com/how-to-set-up-a-ghost-blog-on-a-subdirectory-via-cloudflare-workers-1t2y348r) which uses Cloudflare workers but it wasn't successful most likely because the main domain doesn't point to my server IP like the blog does.

I'm guessing this could potentially be done using the serverless functions in Vercel, but I'm a bit clueless when it comes to that.

I'd appreciate if anyone can give me some ideas about how to go around this or what I can search for over the interwebs 🙏

**Comment by paulogdm** [comment]
Hello,

Check the following: https://nextjs.org/blog/incremental-adoption
